src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorView.xaml

```diff
@@ -0,0 +1,643 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="using:Avalonia.Controls"
+        xmlns:converters="using:Avalonia.Controls.Converters"
+        xmlns:primitives="using:Avalonia.Controls.Primitives"
+        xmlns:pc="clr-namespace:Avalonia.Controls.Primitives.Converters;assembly=Avalonia.Controls.ColorPicker"
+        xmlns:globalization="clr-namespace:System.Globalization;assembly=mscorlib"
+        x:CompileBindings="False">
+
+  <Styles.Resources>
+    <pc:ContrastBrushConverter x:Key="ContrastBrush" />
+    <pc:ThirdComponentConverter x:Key="ThirdComponent" />
+    <converters:ColorToDisplayNameConverter x:Key="ColorToDisplayName" />
+    <converters:ColorToHexConverter x:Key="ColorToHex" />
+    <converters:EnumToBoolConverter x:Key="EnumToBool" />
+    <converters:CornerRadiusFilterConverter x:Key="TopCornerRadiusFilterConverter" Filter="TopLeft, TopRight"/>
+    <converters:CornerRadiusFilterConverter x:Key="BottomCornerRadiusFilterConverter" Filter="BottomLeft, BottomRight"/>
```

Comment:
I think I'm going to pull all the common converters out into a shared file (Fluent.xaml or Default.xaml). There is no reason to declare this in each file separately.

Reply:
You also can define them as static properties and reference with `{x:Static}`.
I.e.
```
public class CornerRadiusFilterConverter : IValueConverter
{
    public static CornerRadiusFilterConverter Instance { get; } = new();
   
    public object? Convert()...
}
```

Reply:
Hm, that's an interesting idea to consider as a new convention. For now will probably keep them in XAML though as that is pretty standard.

Reply:
Pulled out all converters and standardized naming. I see you are also doing the static instance with `BoolConverters.And` for instance.